rviz_common/src/rviz_common/visualization_manager.cpp

```diff
@@ -435,8 +437,8 @@ void VisualizationManager::onUpdate()
   }
 
   frame_count_++;
-
-  if (render_requested_ || wall_dt > 0.01) {
+  constexpr std::chrono::milliseconds ten_ms = std::chrono::milliseconds(10);
+  if (render_requested_ || wall_dt > static_cast<uint64_t>(ten_ms.count())) {
```

Comment:
Oh sorry, you could have used `wall_diff` instead of `wall_dt` as they are the same duration, it's just that `wall_dt` is the "nanoseconds since epoch" version, see:

https://github.com/ros2/rviz/blob/1af13e5676056234fdc177b9052376375829ea1b/rviz_common/src/rviz_common/visualization_manager.cpp#L386

Reply:
I'll wait to hear back from you, I don't mind if you want to leave it or update it to use `wall_diff` (if you do, please make `wall_dt`/`wall_diff` a `const`), but once you tell me I'll run CI again.

Reply:
> I don't mind if you want to leave it or update it to use wall_diff

I'll update it since I'd rather not introduce that variable.